python/pyspark/ml/clustering.py

```diff
@@ -252,7 +252,7 @@ class GaussianMixture(JavaEstimator, _GaussianMixtureParams, JavaMLWritable, Jav
     >>> model.predict(df.head().features)
     2
     >>> model.predictProbability(df.head().features)
-    DenseVector([0.0, 0.4736, 0.5264])
+    DenseVector([0.0, 0.0, 1.0])
```

Comment:
This seems kind of unlikely, as probabilities? is this valid?

Reply:
I had checked it in master, and when maxIter is set to 30, then the model also output `DenseVector([0.0, 0.0, 1.0])`

Reply:
![image](https://user-images.githubusercontent.com/7322292/75656008-01064e00-5c9e-11ea-945c-2c95daf1de2c.png)

The result in `model.transform(df).show()` are not `DenseVector([0.0, 0.0, 1.0])`, but the result in `model.transform(df).head()` is.

![image](https://user-images.githubusercontent.com/7322292/75656118-40cd3580-5c9e-11ea-83aa-a7047fae06d6.png)